### PR TITLE
490

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/490
**Reference person:** @krzysztofmadejski
**Description:**

Added About Us Page

**Unit Tests:**
- `apps/volontulo/tests/views/test_pages.py` modified

Merged already with what @tiriana did.

### CODE REVIEW

File: apps/volontulo/tests/views/test_pages.py [hunk 1]
```diff
@@ -93,3 +93,17 @@ def test__get_regulations_staticpage(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'pages/regulations.html')
         self.assertContains(response, u'Regulamin')
+
+    def test_about_us(self):
+        u"""Test about us."""
+        response = self.client.get('/o-nas')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pages/about-us.html')
+        self.assertContains(response, u'KIM JESTEŚMY?')
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_pages.py:102]: I'm not sure if we really need this check

File: apps/volontulo/tests/views/test_pages.py [hunk 2]
```diff
@@ -93,3 +93,17 @@ def test__get_regulations_staticpage(self):
...
+
+    def test_office_subpage(self):
+        u"""Test office subpage."""
+        response = self.client.get('/office')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pages/office.html')
+        self.assertContains(response, u'Dyżury dla wolontariuszy oraz organizacji')
```
Comment (tomaszwozniak) [inline apps/volontulo/tests/views/test_pages.py]: missing new line at the end file

File: apps/volontulo/templates/pages/about-us.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+        <div>Wolontariat to niesamowita okazja do zdobycia nowych doświadczeń, rozwoju swoich umiejętności i kompetencji, poznania innych ludzi. To także świadomość, że to co robisz ma sens!</div>
+
+        <h3>KIM JESTEŚMY?</h3>
+
+        <div>Wielkopolska Rada Koordynacyjna Związek Organizacji Pozarządowych jest dobrowolnym związkiem organizacji pozarządowych, który został utworzony w 1990 roku z inicjatywy kilkudziesięciu organizacji społecznych. Dziś jesteśmy największym związkiem w Wielkopolsce z zrzeszamy ponad 200 fundacji i stowarzyszeń z całego regionu. Poprzez swoją działalność kształcimy proobywatelskie i proaktywne postawy mieszkańców i mieszkanek Wielkopolski. Jako lider ruchu obywatelskiego w regionie inspirujemy do działań w obszarze „małych ojczyzn”. Uwrażliwiamy na potrzeby i problemy innych. Edukujemy, doradzamy, animujemy, wspieramy zarówno grupy formalne jak i nieformalne oraz niezrzeszonych aktywistów…</div>
```
Comment (filipgorczynski) [inline apps/volontulo/templates/pages/about-us.html:14]: Typo: "Wielkopolsce z zrzeszamy" -> "Wielkopolsce i zrzeszamy"

File: apps/volontulo/templates/pages/about-us.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{% extends "common/col1.html" %}
+
+{% block title %}O nas{% endblock %}
+
+{% block content %}
+    <h2>O Volontulo</h2>
+    <div>
+        <div>Lubisz nowe wyzwania? Chcesz pomagać innym? Zależy Ci na rozwoju? Szukasz wolontariuszy? Jeśli choć na jedno z tych pytań opowiedziałeś/aś twierdząco wolontariat to odpowiedź dla Ciebie! Zgłoś się do Poznańskiego Centrum Wolontariatu prowadzonego przez Wielkopolską Radę Koordynacyjną Związek Organizacji Pozarządowych. Naszym celem jest pomoc we wzajemnym odnalezieniu się ludzi, którzy chcą realizować się jako wolontariusze/szki oraz organizacji i instytucji, które takich osób poszukują.</div>
```
Comment (filipgorczynski) [inline apps/volontulo/templates/pages/about-us.html:8]: Shouldn't this be wrote as `<p>` element - as paragraph?

File: apps/volontulo/tests/views/test_newsletter.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+u"""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:3]: remove u""

File: apps/volontulo/tests/views/test_newsletter.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+.. module:: test_newsletter
+"""
+from django.test import TestCase
+
+
+class TestAboutus(TestCase):
+    u"""Class responsible for testing newsletter specific views."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:10]: Remove u""

File: apps/volontulo/tests/views/test_newsletter.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+    # pylint: disable=invalid-name
+    def test__newsletter(self):
+        u"""Test getting newsletter signup page as anonymous."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:14]: remove u""

File: apps/volontulo/tests/views/test_pages.py [hunk 8]
```diff
@@ -93,3 +93,18 @@ def test__get_regulations_staticpage(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'pages/regulations.html')
         self.assertContains(response, u'Regulamin')
+
+    def test_about_us(self):
+        u"""Test about us."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_pages.py:98]: Remove u""

File: apps/volontulo/tests/views/test_pages.py [hunk 9]
```diff
@@ -93,3 +93,18 @@ def test__get_regulations_staticpage(self):
...
+        response = self.client.get('/o-nas')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pages/about-us.html')
+        self.assertContains(response, u'KIM JESTEŚMY?')
+
+    def test_office_subpage(self):
+        u"""Test office subpage."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_pages.py:105]: Remove u""

File: apps/volontulo/tests/views/test_pages.py [hunk 10]
```diff
@@ -93,3 +93,18 @@ def test__get_regulations_staticpage(self):
...
+        response = self.client.get('/office')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pages/office.html')
+        self.assertContains(response,
+                            u'Dyżury dla wolontariuszy oraz organizacji')
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_pages.py:110]: Remove u""

File: apps/volontulo/views/__init__.py [hunk 11]
```diff
@@ -264,6 +264,17 @@ def contact_form(request):
     )
 
 
+def newsletter_signup(request):
+    u"""Newsletter signup page
```
Comment (filipgorczynski) [inline apps/volontulo/views/__init__.py:264]: Remove u""

**Comment by KrzysztofMadejski** [comment]
@magul

**Pull request merged**